### ISSUE TITLE
Attempting to insert a vacation day throws errors

### ISSUE THREAD

**Issue by pavulon18** [pavulon18/gcas_timesheet_php_website#64]
**Describe the bug**
An employee was attempting to enter a vacation day onto his timesheet.  Several errors were returned.  He sent me a screenshot.  I'll try to upload it when I get a chance.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by pavulon18** [comment]
I think I know what happened here and why this is happening.  I believe I was trying to get all of my variable names under one naming convention when I had to stop development because of class.  When class was finished and I started looking at the code again, I could not recall exactly what I was doing or how far I had gotten.

I will need to work through all of the pages looking for error messages and fixing them as I go.  This should eliminate this problem.

I think there was a very specific reason as to why I was changing the naming conventions, but as of right now I cannot recall why I was doing it.  I guess I will eventually figure it out.
